.github/workflows/main.yml

```diff
@@ -8,20 +8,41 @@ on:
 jobs:
   build_all_packages:
     runs-on: ubuntu-18.04
+    container:
+      image: archlinux/archlinux:latest
+      options: --privileged
```

Comment:
Instead of `--privileged`, does it work with `--security-opt=seccomp=unconfined`? If yes, I prefer this to be used. And a comment can be added in order to add a reference to https://github.com/archlinuxhardened/selinux/issues/82

Reply:
Indeed it works. I thought it didn't but I had wrong options there.

Reply:
Oh and reference to #82 is on the first comment already :)